public/static/locales/en-US/classes.json

```diff
@@ -54,6 +54,11 @@
       "pantsSize": "School pants size",
       "elective": "Chosen elective"
     },
+    "filterColumns": {
+      "label": "Filters",
+      "noElective": "Without elective",
+      "hasAllergies": "Allergies"
+    },
```

Comment:
I’m not the biggest fan of this name change. I think `columns` is good as is, and `filterColumns` makes no sense. We are filtering rows, not columns, and I think `filters` on its own communicate enough. Try to be succinct.